internal/command/machine/run.go

```diff
@@ -160,21 +166,73 @@ func newRun() *cobra.Command {
 		sharedFlags,
 	)
 
-	cmd.Args = cobra.MinimumNArgs(1)
+	// If we pass a JSON machine configuration, we don't build a local image.
+	// That introduces this ugliness, which changes the expected arg count depending
+	// on flags passed.
```

Comment:
this complexity is perhaps a good sign that this functionality should live in a different command. How about moving this feature to something like `fly m create` or `fly m create_in_api` or something like that?

Reply:
a totally different idea:

```
fly machine api post
```

so it's super clear this is simply a passthrough to the machines api, and won't get confused with the other machine sub commands.

In the future, we could add other methods too. Like `fly machine api put`.

Reply:
or maybe `fly machine flaps-api post`, to maybe be less likely confused with the api-proxy subcommand

Reply:
How about `json-api`? flaps feels more like an implementation detail imo